runtime/rasdump/javadump.cpp

```diff
@@ -1468,19 +1468,91 @@ JavaCoreDumpWriter::writeEnvUserArgsHelper(J9VMInitArgs *vmArgs)
 	}
 
 	{
-		/* write ignored options */
-		bool anyIgnored = false;
+		/* Write ignored options.
+		 *
+		 * An option is printed as ignored if it has been consumed or if it is an -Xjit or
+		 * -Xaot option that fall under the rules below.
+		 *
+		 * Ignored options for -Xjit and -Xaot follows these rules:
+		 * 1. If -Xjit is followed by -Xint or -Xnojit,
+		 *    then any -Xjit:optionString beforehand appears as ignored.
+		 * 2. If -XX:+MergeCompilerOptions is present,
+		 *    then all -Xjit options after -Xint or -Xnojit should not be ignored.
+		 * 3. If multiple -Xjit strings appear, all but the last one will appear as ignored.
+		 *
+		 * Similar rules apply for -Xaot.
+		 */
+
+		/* The indices for the relevant options in args->nOptions. */
```

Comment:
I don't see how "in args->nOptions" makes sense here (it's an integer, not something that could contain options).

Reply:
Please improve this comment. I suggest simply removing "in args->options" unless I misunderstand its intent.